docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1535 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from matplotlib.ticker import MaxNLocator
</pre><p>This is not used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
ah, yeah dropped another example that I ended up not liking. Good catch.